src/com/flower/kiev/FlowerStore.java

```diff
@@ -0,0 +1,79 @@
+package com.flower.kiev;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FlowerStore {
+
+    private int wallet;
+    private Flower[] flowers;
+
+    public int getWallet() {
+        return wallet;
+    }
+
+    public void printFlowers() {
+        for (int i = 0; i < flowers.length; i++) {
+            System.out.print(flowers[i].getClass().getSimpleName() + " ");
+        }
+    }
+
+    public Flower[] sell(int numRose, int numChamomile, int numTulip) {
+        int numOfFlowers = numRose + numChamomile + numTulip;
+        flowers = new Flower[numOfFlowers];
+
+        for (int i = 0; i < numRose; i++) {
+            flowers[i] = new Rose();
+            wallet += new Rose().getFLOWER_PRICE();
+        }
+
+        for (int i = 0; i < numChamomile; i++) {
+            flowers[numRose + i] = new Chamomile();
+            wallet += new Chamomile().getFLOWER_PRICE();
+        }
+
+        for (int i = 0; i < numTulip; i++) {
+            flowers[numRose + numChamomile + i] = new Tulip();
+            wallet += new Tulip().getFLOWER_PRICE();
+        }
+
+        for (int i = 0; i < flowers.length; i++) {
```

Comment:
Why did you print information here? It should not be done in this method. 
We should use SOLID principle (S- single responsibility). 
The responsibility of this method just add flowers to the array and return it but not print information.